src/api.ts

```diff
@@ -72,7 +72,7 @@ router.post('/nft-claimer/deploy', async (req, res) => {
       })
     );
   } catch (e: any) {
-    capture(e, { context: { body: req.body } });
+    capture(e, { contexts: { body: { body: req.body } } });
```

Comment:
let's simplify the calling of this and move the nested structure of the second argument inside the sentry wrapper

Reply:
By doing this, we lose the ability to switch from `contexts` to anything else